lib/OpenQA/Utils.pm

```diff
@@ -903,4 +903,6 @@ sub fix_top_level_help { @ARGV = () if ($ARGV[0] // '') =~ qr/^(-h|(--)?help)$/
 
 sub looks_like_url_with_scheme { return !!Mojo::URL->new(shift)->scheme }
 
+
+sub indent_log_raw_output { @ARGV = () if ($ARGV[0] // '') =~ s/\r\n/,\s\s/gm;}
```

Comment:
I suggest to not add a function here when it is only used in a single location. OpenQA::Utils is not a dumping ground. Just move the function back into the class where used.

The name is fine, the argument is wrong :)